lib/generators/decidim/templates/social_share_button.rb

```diff
@@ -0,0 +1,3 @@
+SocialShareButton.configure do |config|
```

Comment:
I'd add a comment with a link to the official docs so people know how to configure it